src/pks/flow/constitutive_relations/thaw_depth/column_water_content_evaluator.cc

```diff
@@ -18,11 +18,11 @@ ColumnWaterContentEvaluator::ColumnWaterContentEvaluator(Teuchos::ParameterList&
     : SecondaryVariableFieldEvaluator(plist)
 {
   domain_ = Keys::getDomain(my_key_);
-  auto pos = domain_.find_last_of('_');
+  auto pos = domain_.find_last_of(':');
```

Comment:
Whups, one more place to clean up and remove hard-coded :

Reply:
fixed